cypress/README.md

```
@@ -27,19 +27,19 @@ All these commands require the relevant env vars to be exported in the terminal
 To launch the Cypress test runner:
 
 ```
-yarn cypress-ui
+npm run cypress-ui
 ```
 
 (Note this shows all tests; you may want to append e.g. `--env grepTags=ci` to see just one subset)
 
 To run Cypress in headless mode:
 
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)